feature/session/src/main/res/layout/fragment_sessions.xml

```diff
@@ -21,11 +21,13 @@
                 style="@style/Widget.DroidKaigi.Backdrop.BackLayer"
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
+                android:fillViewport="true"
                 >
 
                 <androidx.constraintlayout.widget.ConstraintLayout
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
+                    android:paddingBottom="300dp"
```

Comment:
👀

Reply:
I think you can calculate that size

1. please export peekhight to dimens
app:behavior_peekHeight="64dp"
2. And use insetter in SessionsFragment
I think you can calc sum 64dp + window inset bottom

Reply:
thanks!